<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSettingsWidget</class>
 <widget class="QWidget" name="AdvancedSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>358</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Logging</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Log Level:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="logLevel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="logChannels">
        <property name="toolTip">
         <string>Log Channels</string>
        </property>
        <property name="icon">
         <iconset theme="filter-line"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="logToConsole">
          <property name="text">
           <string>Log To System Console</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="logToWindow">
          <property name="text">
           <string>Log To Window</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="logToDebug">
          <property name="text">
           <string>Log To Debug Console</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="logToFile">
          <property name="text">
           <string>Log To File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>System Settings</string>
     </property>
     <layout class="QGridLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="showDebugMenu">
        <property name="text">
         <string>Show Debug Menu</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Tweaks/Hacks</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTableWidget" name="tweakOptionTable">
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Option</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="resetToDefaultButton">
        <property name="text">
         <string>Reset To Default</string>
        </property>
        <property name="icon">
         <iconset theme="restart-line"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/duckstation-qt.qrc"/>
 </resources>
 <connections/>
</ui>
